framework/test/mocha/unit/modules/chain/submodules/loader.js

```diff
@@ -455,286 +379,4 @@ describe('loader', () => {
 			});
 		});
 	});
-
-	describe('__private.loadBlocksFromNetwork', () => {
```

Comment:
Why do we remove this instead of adapting it to the new times?

Reply:
These unit tests and the underlying logic were poorly implemented. This refactoring improves this a bit but it's still not in a sufficiently clean/settled state to be unit tested.

Reply:
So how do you assure it's working well?